src/scanner/mod.rs

```diff
@@ -48,35 +49,40 @@ impl Scanner {
     /// Returns all open ports as Vec<u16>
     pub async fn run(&self) -> Vec<SocketAddr> {
         let ports: Vec<u16> = self.port_strategy.order();
-        let batch_per_ip: usize = self.batch_size as usize / self.ips.len();
+        // let batch_per_ip: usize = self.batch_size as usize / self.ips.len();
```

Comment:
Delete code that is not needed instead of commenting it out :)

Reply:
It's gone.